app/models/report_generators/pit/fy2018/base.rb

```diff
@@ -845,7 +845,7 @@ def chronic_scope
 
     def sh_cols
       @sh_cols ||= {
-        project_type: act_as_project_overlay,
+        project_type: she_s[:project_type].as('client_id'),
```

Comment:
```suggestion
        project_type: she_s[:project_type].as('project_type'),
```

Reply:
Thanks for spotting! I had the wrong table alias too 🤦